src/main/java/org/spongepowered/api/world/World.java

```diff
@@ -302,4 +304,9 @@
     @Override
     MutableBlockVolumeWorker<? extends World> getBlockWorker();
 
+    @Override
+    default Text toText() {
```

Comment:
Don't randomly implement it for one interface just because Minecraft has no implementation for it internally. This should be in SpongeCommon